Create DS Code Documentation.md

armaf002 opened the pull request:
General outline and explanation of the code from the data science team

armaf002 committed:
Create DS Code Documentation.md

General outline and explanation of the code from the data science team

armaf002 committed:
Add files via upload

The project code in the ipynb format from the data science team and the trained model that has been saved in a serialized format for the deployment.

armaf002 committed:
Delete Plotly.ipynb

There was an error in the code

armaf002 reviewed (commented):
This is not accurate, I will send new one

armaf002 committed:
Delete model.pkl

Error in the file

armaf002 committed:
Add files via upload

These are the correct ones

krissemmy reviewed (changes requested):
can you change the notebook name to show it involve the data science aspect

armaf002 commented:
Alright

armaf002 committed:
Rename Plotly (1).ipynb to Data Science aspect.ipynb

The document as renamed from Rename Plotly (1).ipynb to Data Science aspect.ipynb
